src/components/Restaurant.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
```

Comment:
react files - are normaly denoted as .jsx

Also notice that the defined components should be capitalised(name of component and file name) this is also how react understands if the component is standard or custom/user made.